TappyReader/SerialTappy.swift

```diff
@@ -24,15 +24,15 @@ import Foundation
 @objc
 public class SerialTappy : NSObject, Tappy {
     
-    var communicator : TappySerialCommunicator
+    @objc var communicator : TappySerialCommunicator
```

Comment:
I wonder if this is still needed if i add the `@objc` annotation to the `getCommunicator` method?